src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/TagHelperDescriptorExtensions.cs

```diff
@@ -9,6 +9,16 @@ namespace Microsoft.AspNetCore.Razor.Language.Components
 {
     internal static class TagHelperDescriptorExtensions
     {
+        public static bool IsAnyComponentDocumentTagHelper(this TagHelperDescriptor tagHelper)
```

Comment:
What be a ComponentDocumentTagHelper and why do we now need this? I know you're running out of names and that this is required, i just want to make sure I 100% understand what it means and why.

Reply:
Is it any kind of tag helper that should show up in a component document.

Reply:
I see I see 👍